### PR TITLE
feat: health widget

### PR DESCRIPTION
**Pull request by sjaanus**
image

### CODE REVIEW

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+                <Typography variant='body2'>
+                    On average, your project health has remained at {health}%
+                    the last 4 weeks
+                </Typography>
+            </ChartRow>
+            <DescriptionText variant='body2'>
+                Remember to archive your stale feature flags to keep the project
+                health growing
+            </DescriptionText>
+            {isOss() && <Link to='/insights'>View health over time</Link>}
```
Comment (sjaanus) [inline frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:90]: OSS does not have insights.

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                        fontSize='16px'
+                        fontWeight='bold'
+                    >
+                        {health}%
+                    </text>
+                </StyledSVG>
+                <Typography variant='body2'>
+                    On average, your project health has remained at {health}%
+                    the last 4 weeks
+                </Typography>
```
Comment (thomasheartman) [inline frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:84]: I'm assuming that you'll work and make it dynamic in a future update?

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                        x='50'
+                        y='50'
+                        textAnchor='middle'
+                        dominantBaseline='middle'
+                        fill={theme.palette.text.primary}
+                        fontSize='16px'
+                        fontWeight='bold'
+                    >
+                        {health}%
+                    </text>
```
Comment (thomasheartman) [inline frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:79]: Comparing to the sketches, I think the health % text is a little small. I also don't think it should be bold.

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+                </StyledSVG>
+                <Typography variant='body2'>
```
Comment (thomasheartman) [inline frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:81]: Do you need the variant here at all? I think body2 has a font size of 12px, while the default (body1) uses 14px. The sketches also use 14 px.
```suggestion
                <Typography >
```

**Review by thomasheartman — APPROVED** [review]
Looking good! Couple minor things that can be ironed out later.

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx (hunk 2 above)
Comment (sjaanus) [reply frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:84]: Yes

File: frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx (hunk 4 above)
Comment (sjaanus) [reply frontend/src/component/project/Project/ProjectStatus/ProjectHealth.tsx:81]: Body2 is 14px and body1 is 16px.

**Pull request merged**